rca/people/models.py

```diff
@@ -350,10 +374,30 @@ def get_area_linked_filters(self):
                 expertise.append({"title": i.area_of_expertise.title})
         return expertise
 
+    def get_Directorate_linked_filters(self):
+        """ For the expertise taxonomy thats listed out in key details,
+        they need to link to the parent staff picker page with a filter pre
+        selected"""
+
+        parent = self.get_parent()
+        directorate = []
```

Comment:
small suggestion, as this is a list I think `directorates` would make more sense as a variable name